reference/marketplace-api.yaml

```diff
@@ -1198,6 +1198,9 @@ paths:
                 account_id:
                   type: string
                   description: A unique ID that can be used to identify an account.
+                item_id:
+                  type: string
+                  description: Field for specifying the add-on ID if the change request is for an add-on.
```

Comment:
Why would a vendor need to specify the add-on id when calling the resolve change request endpoint? Can't you look that up from the change_request_id?

Reply:
The endpoint needs to know which app or add-on to resolve the change spend request for. For apps, we are able to get the appID stored on the context, but if we are resolving a spend request for an add-on, we only have the parent appID stored on the context so we need to provide the addonID as an argument.